Include/internal/pycore_stackref.h

```diff
@@ -262,6 +265,14 @@ PyStackRef_UntagInt(_PyStackRef i)
 }
 
 
+static inline _PyStackRef
+PyStackRef_IncrementTaggedIntNoOverflow(_PyStackRef ref)
+{
+    assert(ref.bits != (uintptr_t)-1); // Deosn't overflow
```

Comment:
I don't understand why you use this condition. Should it not be `assert(ref.bits + 4 > ref.bits)` or something like that?